test/converters/BindingConverters.test.ts

```diff
@@ -7,12 +7,11 @@ import { fromString } from 'long';
 import 'mocha';
 import { getBindingDefinitions, getNormalizedBindingData } from '../../src/converters/BindingConverters';
 import { fromTypedData } from '../../src/converters/RpcConverters';
-import { toRpcHttp } from '../../src/converters/RpcHttpConverters';
 import { FunctionInfo } from '../../src/FunctionInfo';
 
 describe('Binding Converters', () => {
     it('normalizes binding trigger metadata for HTTP', () => {
-        const mockRequest: RpcTypedData = toRpcHttp({ url: 'https://mock' });
+        const mockRequest: RpcTypedData = { http: { url: 'https://mock' } };
```

Comment:
This was the only instance in the code where `toRpcHttp` was used to convert an HTTP request rather than an HTTP response. I changed it in favour of having stricter type enforcement